tequilapi/endpoints/service.go

```diff
@@ -164,11 +175,40 @@ func (se *ServiceEndpoint) ServiceStart(resp http.ResponseWriter, req *http.Requ
 		return
 	}
 
-	se.serviceState = service.Running
+	optionsParser, ok := se.optionsParser[sr.ServiceType]
+	if !ok {
+		utils.SendErrorMessage(resp, "Invalid service type", http.StatusBadRequest)
+		return
+	}
+
+	options, err := optionsParser(sr.Options)
+	if err != nil {
+		utils.SendErrorMessage(resp, "Invalid options", http.StatusBadRequest)
+		return
+	}
+
+	for _, instance := range se.serviceManager.List() {
+		proposal := instance.Proposal()
+		if proposal.ProviderID == sr.ProviderID && proposal.ServiceType == sr.ServiceType {
+			utils.SendErrorMessage(resp, "Service already running", http.StatusConflict)
+			return
+		}
+	}
+
+	if err := se.identityManager.Unlock(sr.ProviderID, sr.Passphrase); err != nil {
```

Comment:
Not sure if we need to do unlock automatically, or force a user to request unlock before starting services.

Reply:
I think the auto unlock is a good option, but we need appropriate handling to let the user know if the unlock failed because the passphrase provided was invalid or there was no passphrase provided

Reply:
I would like to follow atomic action in `POST /service` endpoint, same as we have in `POST /connection`.
That means caller of endpoint should care about identify unlocking.

e.g. CLI command `myst service` does such actions:
- lists current identities
- Autoselect provider identity and unlock it
- Call Service start endpoint

Reply:
Removed auto unlock for Tequilapi endpoint. We are getting an explicit error if identity was not unlocked.